backend/fms_core/models/index_structure.py

```diff
@@ -0,0 +1,29 @@
+import reversion
+
+from django.db import models
+
+from .tracked_model import TrackedModel
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._validators import name_validator
+
+__all__ = ["IndexStructure"]
+
+@reversion.register()
+class IndexStructure(TrackedModel):
+    name = models.CharField(unique=True, max_length=STANDARD_NAME_FIELD_LENGTH, validators=[name_validator],
+                            help_text="The name of the index structure.")
+
+    flanker_3prime_forward = models.ForeignKey("Sequence", related_name="flanker_3prime_forward",
+                                               on_delete=models.PROTECT, help_text="Flanker on the 3' forward direction")
```

Comment:
using single quote in the code is a bit risky. Might be safer to spell it as "prime".